components/x-audio/src/redux/__tests__/player-logic.test.js

```diff
@@ -59,22 +59,16 @@ describe('middleware', () => {
 		return { store, next, invoke, audio };
 	}
 
-	test('requestPlay with url sets src and plays', () => {
+	test('loadMedia set the URL and configures tracking', () => {
```

Comment:
Pedant comment but worth just changing the name of this test in the tracking work?